docs/building-spark.md

```diff
@@ -27,7 +27,7 @@ license: |
 ## Apache Maven
 
 The Maven-based build is the build of reference for Apache Spark.
-Building Spark using Maven requires Maven 3.8.7 and Java 8.
+Building Spark using Maven requires Maven 3.8.8 and Java 8/11/17.
```

Comment:
Why do you touch `8/11/17` in this PR? Could you remove it from this PR, @pan3793 ?

Reply:
https://github.com/apache/spark/pull/22781 changed the content from "Java 8+" to "Java 8" because

> Since we didn't test Java 9 ~ 11 up to now in the community

Currently, the GA tests building Spark using Maven w/ Java 8/11/17, so change it accordingly

Reply:
It makes sense. Thanks!